lib/ex_quiz_it_web/live/quiz_live.ex

```diff
@@ -3,23 +3,37 @@ defmodule ExQuizItWeb.QuizLive do
   Main LiveView for the quiz
   """
   use Phoenix.LiveView, layout: {ExQuizItWeb.LayoutView, "live.html"}
+  alias ExQuizIt.Questions.Enums
 
-  @questions ["what does the fox say?", "what did the confident pickle say?", "are we there yet?"]
+  @questions Enums.q_and_a()
 
   @impl Phoenix.LiveView
   def mount(_params, _session, socket) do
-    question = Enum.random(@questions)
-    {:ok, assign(socket, message: "Make a guess:", question: question, answer: "")}
+    q_and_a = Enum.random(@questions)
+
+    {:ok,
+     assign(socket,
+       message: "Make a guess:",
+       q_and_a: q_and_a,
+       question: Map.get(q_and_a, "question"),
+       answer: ""
+     )}
```

Comment:
What if instead of setting the answer to an empty string to start, you just set the answer and use a boolean for whether to show it or not

That way you only need two variables: `q_and_a` and `show_answer`

Reply:
Done!